tests/test_send_file.py

```diff
@@ -115,10 +118,11 @@ def test_non_ascii_name(name, ascii, utf8):
     content_disposition = rv.headers["Content-Disposition"]
     assert f"filename={ascii}" in content_disposition
 
+    """Per RFC 5987/8187, "extended" values may *not* be quoted.
```

Comment:
I think this would be better as a comment `#`, although I don't think it is appropriate here.

Reply:
Would you prefer I remove the comment altogether or reformat it with single-line comment(s)? It was largely a reminder to myself about which standards docs provided the basis for the changes. If it doesn't seem helpful, I'm happy to pull it out entirely.

Reply:
Could you pull it out entirely. The comment is in the commit and the docstring above where it is more relevant in my view.

Reply:
Done!